src/rpc/cli.cpp

```diff
@@ -231,6 +231,7 @@ static int cli_quitting = false;
 /**
  * Get next character from stdin, or EOF if got a SIGINT signal
  */
+#ifndef WIN32
```

Comment:
I think it's better to create a separate PR for this change. By the way, move this up a few lines so the comments will be `ifndef`ed as well?

Reply:
Done. See PR #132